Products/DataCollector/ApplyDataMap.py

```diff
@@ -10,9 +10,14 @@
 
 import sys
 from collections import defaultdict
+import urlparse
+
 import logging
 log = logging.getLogger("zen.ApplyDataMap")
 
+import requests
+session = requests.Session()
```

Comment:
Should we wrap this and the method handle_datamap in a class so we have all the zing datamap forwarding logic encapsulated?

Reply:
sure